examples/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  db-postgres:
+    image: postgres
+    command: postgres -c log_statement=all
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=examples
+
+  aiohttp:
+    build:
+      context: ..
+      dockerfile: examples/Dockerfile
+    depends_on:
+      - db-postgres
+    ports:
+      - 127.0.0.1:10080:10080
+    volumes:
+      - ../examples:/app/examples
```

Comment:
Why not just put peewee_async in requirements? You want to use the example in the development proccess of peewee-async?

Reply:
Found your comment this is for live-reload while developing. I would not use docker for the application only for database The docker config for database already exists. The application can be run by bash command